src/platforms/rcore_drm.c

```diff
@@ -2480,82 +2655,4 @@ static int FindNearestConnectorMode(const drmModeConnector *connector, uint widt
     return nearestIndex;
 }
 
-// Compute framebuffer size relative to screen size and display size
```

Comment:
This function is in raylib current master branch https://github.com/raysan5/raylib/blob/master/src/platforms/rcore_drm.c#L2485